.gitignore

```diff
@@ -50,3 +50,6 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+
+.vscode
```

Comment:
This belongs to your global .gitignore. The project .gitignore should only contain language specific ignores.

Reply:
Cleaned up.